enterprise/coderd/license/license.go

```diff
@@ -87,7 +87,7 @@ func Entitlements(
 		if isTrial {
 			showWarningDays = 7
 		}
-		isExpiringSoon := daysToExpire > 0 && daysToExpire < showWarningDays
+		isExpiringSoon := isTrial && daysToExpire > 0 && daysToExpire < showWarningDays
```

Comment:
This is probably dangerous since it won't show to admins if the license is expiring

Reply:
Correct. The change should probably just be made on the CLI to only display if you're an admin.

Reply:
@coadler i'm not sure how to build this into the CLI. can you take this on?

Reply:
Would probably be best to make an issue and prioritize into a sprint, I don't have the cycles currently.